validator/client/attest.go

```diff
@@ -105,15 +114,6 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		Signature:       sig,
 	}
 
-	indexedAtt.Signature = sig
-	if err := v.postAttSignUpdate(ctx, indexedAtt, pubKey, signingRoot); err != nil {
```

Comment:
We were doing slashing protection checks twice in an attempt to solve race condition. Instead, we only do it once above